k8s/perftest/common/ccpay/ccpay-bulkscanning-app.yaml

```diff
@@ -0,0 +1,42 @@
+---
+apiVersion: flux.weave.works/v1beta1
+kind: HelmRelease
+metadata:
+  name: ccpay-bulkscanning-api
+  namespace: fees-pay
+  annotations:
+    flux.weave.works/automated: "true"
+    flux.weave.works/tag.java: glob:prod-*
+spec:
+  releaseName: ccpay-bulkscanning-api
+  rollback:
+    enable: true
+  chart:
+    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
+    name: ccpay-bulkscanning-api
+    version: 0.0.4
+  values:
+    java:
+      replicas: 2
+      useInterpodAntiAffinity: true
+      applicationInsightsInstrumentKey: "280b07a1-f1a1-4369-8e21-0bb2c2995f1e"
```

Comment:
Please check the app insights key here as I don't think this is the right key. This seems like the app insights key for AAT.

Reply:
I think it is `724b2f84-235d-4448-adff-5fa1dc1b3864`